### DISCUSSION TITLE
/vscode in Copilot Chat returns an error

### DISCUSSION THREAD

**Discussion by vuyp** [community/community#56155] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Everything else is fine, but when using the '/vscode' command the Indexing message appears and then the error immediately after: `Error: ENOENT: no such file or directory, unlink '/home/codespace/.vscode-remote/data/User/globalStorage/github.copilot-chat/settingEmbeddings.json'`.
**NOTE - This is in Github Codespaces. Rebuilding the Codespace, removing and creating a new one and trying again, and switching to Stable did not work.**

![image](https://github.com/community/community/assets/113992401/f91c6471-b3dc-48a7-8d94-1affe90b1e3b)

**Comment by RobBoeckermann** [comment]
Same issue here. The file does not exist and the directory only contains `api.json`:

**Comment by sarimarton** [comment]
Same here


```
Error: ENOENT: no such file or directory, unlink '/home/sarim/.vscode-server-insiders/data/User/globalStorage/github.copilot-chat/commandEmbeddings.json'
```

**Comment by scriptogre** [comment, marked as answer]
Hey! I managed to fix this issue by simply running VSCode Insiders as administrator. 🤦

**Reply by vuyp** [reply]
Well...I'm using codespaces, so it isn't like I can fix it by doing that. As mentioned in the post. But thanks for finding a way to fix for people on desktop!

**Reply by vuyp** [reply]
Anyway, marked as answer BUT it isn't really - this is still a problem for Copilot on Codespaces which needs to be fixed!!

**Reply by vuyp** [reply]
Nevermind, it seems to have been fixed after a Copilot update. Thanks!
